nvflare/ha/overseer/utils.py

```diff
@@ -33,6 +33,20 @@
     from .mem_store import get_all_sp, get_primary_sp, get_sp_by, update_sp  # noqa
 
 
+system_state = "ready"
```

Comment:
If we re-write this file as a class, the system state can be treated as class variable?

Reply:
Yes.  But right now, I think it's better to keep it as simple as possible.  Later, if we need to add more functionalities into overseer, we can start to refactor its base utils into different classes.